frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -667,6 +672,24 @@ const plugin: Plugin<ConsumedExtensions> = [
       required: [OCS_FLAG],
     },
   },
+  // In case Noobaa is deployed without OCS Operator
+  {
+    type: 'NavItem/Href',
+    properties: {
+      id: 'ocsdashboards',
```

Comment:
```suggestion
      id: 'ocs-dashboards',
```

Reply:
Let's keep it same https://github.com/openshift/console/pull/8669/files#diff-10b74cbaa04568da87a8244d0807997527af2aa4bf767d6ca29133ec4713eb48R662